clang/test/Sema/precision-and-size-conversion.c

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -fsyntax-only -verify=precision-loss,precision-gain,size-change -Wimplicit-float-conversion -Wdouble-promotion -Wimplicit-float-size-conversion \
+// RUN:   -triple x86_64-apple-darwin %s
```

Comment:
For my own understanding, what is the significance of this triple?  Why are we using this instead of say, ``x86_64-pc-windows-msvc`` that we typically see on sycl host?

Reply:
There is no significance behind chosing the darwin target. It could as well be windwos-msvc and the test would pass. 
I just picked it up from some tests I referenced and felt adventurous to go with it!